deployments/gpu-operator/templates/clusterpolicy.yaml

```
@@ -511,8 +511,9 @@ spec:
     {{- if .Values.dcgmExporter.args }}
     args: {{ toYaml .Values.dcgmExporter.args | nindent 6 }}
     {{- end }}
-    {{- if .Values.dcgmExporter.config }}
-    config: {{ toYaml .Values.dcgmExporter.config | nindent 6 }}
+    {{- if and (.Values.dcgmExporter.config) (.Values.dcgmExporter.config.name) }}
+    config:
+      name: {{ .Values.dcgmExporter.config.name }}
```

Aren't we forgetting to add `config.create` and `config.data`? Did you mean the below?
```suggestion
    config: {{ toYaml .Values.dcgmExporter.config | nindent 6 }}
```

---

Oh disregard my comment, these are not relevant for clusterpolicy.